### ISSUE TITLE
Influx error

### ISSUE THREAD

**Issue by jdk-sal3** [renasboy/crypto-trader-bot#1]
~/crypto-trader-bot$ tail -f log/bot.log
    algo_helper = influx_algo_helper("listener", EXCHANGE, SYMBOL_1, SYMBOL_2)
  File "/crypto-trader-bot/src/influx_algo_helper.py", line 21, in __init__
    self.prev_macd = self.macd
  File "/crypto-trader-bot/src/influx_algo_helper.py", line 170, in macd
    result = self.influx.query(query_macd)
  File "/usr/local/lib/python3.10/site-packages/influxdb/client.py", line 521, in query
    response = self.request(
  File "/usr/local/lib/python3.10/site-packages/influxdb/client.py", line 378, in request
    raise InfluxDBClientError(err_msg, response.status_code)
influxdb.exceptions.InfluxDBClientError: 401: {"code":"unauthorized","message":"Unauthorized"}

**Comment by bigbigbigfish** [comment]
The same issue, what is root case?

**Comment by ultult** [comment]
any updates on this?

**Comment by renasboy** [comment]
yeah maybe the influx version needs to be pinned inside compose template file to 1.8 instead of latest, just sent a little patch, hope it works for you, thanks for noticing!
